galaxy.yml

```diff
@@ -50,10 +45,10 @@
       restart_galaxy: true
       when: install_galaxy
       
-    #check if tool-data persisted on galaxy (it does nothing if galaxy_data == galaxy_server_dir)
     - role: galaxy.movedata
-      check_tool_data: yes
-      when: install_galaxy
+      move_to_persistent: true
```

Comment:
take a look here to understand how to test different configurations:
https://github.com/ARTbio/ansible-artimed/blob/master/.travis.yml#L32